compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -10142,7 +10142,7 @@ OMR::Z::CodeGenerator::addVMThreadPreCondition(TR::RegisterDependencyConditions
       //TODO: using addPreConditionIfNotAlreadyInserted() is slower than using addPreCondition()
       //think about changing the name of this method to addVMThreadPreConditionIfNotAlreadyInserted()
       //and add one for addVMThreadPreCondition, which uses deps->addPreCondition()
-      deps->addPreConditionIfNotAlreadyInserted(reg,(TR::RealRegister::RegNum)self()->getVMThreadRegister()->getAssociation());
+      deps->addPreConditionIfNotAlreadyInserted(reg,(TR::RealRegister::RegDep)self()->getVMThreadRegister()->getAssociation());
```

Comment:
Space after 'reg,' needed. Also we should avoid C style casts and use `static_cast` instead. Once we move to scoped enums you won't have to search for these types of casts as they'll be compile time errors (if using `static_cast` - not if using unsafe C style cast). Similarly elsewhere.

Reply:
Done: a2e8dd2